main/lsp/requests/formatting/document_formatting.cc

```diff
@@ -0,0 +1,59 @@
+#include "main/lsp/requests/formatting/document_formatting.h"
```

Comment:
ooc, why did you create a `formatting/` folder? this would be the first folder inside `main/lsp/requests/` and it seems somewhat redundant to me, especially now that we can also delete the `*_noop.cc` as mentioned in the other comment. I'd love if we could put this back in the place it was before 🙏